resources/public/api/conf/3.0/lossClaimType_amend.raml

```diff
@@ -0,0 +1,32 @@
+is:
+  - headers.acceptJson
+  - headers.testScenarioHeader
+  - errors.formatNino
+  - errors.formatClaimId
+  - errors.formatTypeOfClaim
+  - errors.ruleTypeOfClaim403
```

Comment:
RULE_TYPE_OF_CLAIM_INVALID has been updated in the spec and can be changed to ruleTypeOfClaim400